projects/1.16/assets/thermal_extra/thermal_extra/lang/zh_cn.json

```
@@ -0,0 +1,83 @@
+  "item.thermal_extra.vukaium_bucket": "乌卡桶",
+  "item.thermal_extra.vukaium_coin": "乌卡币",
+  "item.thermal_extra.vukaium_ingot": "乌卡锭",
+  "item.thermal_extra.vukaium_dust": "乌卡粉",
+  "item.thermal_extra.vukaium_gear": "乌卡齿轮",
+  "item.thermal_extra.vukaium_nugget": "乌卡粒",
+  "item.thermal_extra.vukaium_plate": "乌卡板",
+  "fluid.thermal_extra.nebulium": "星云",
+  "item.thermal_extra.nebulium_bucket": "星云桶",
+  "item.thermal_extra.nebulium_coin": "星云币",
+  "item.thermal_extra.nebulium_ingot": "星云锭",
+  "item.thermal_extra.nebulium_dust": "星云粉",
+  "item.thermal_extra.nebulium_gear": "星云齿轮",
+  "item.thermal_extra.nebulium_nugget": "星云粒",
+  "item.thermal_extra.nebulium_plate": "星云板",
+  "fluid.thermal_extra.nectium": "粘液康铜",
+  "item.thermal_extra.nectium_bucket": "粘液康铜桶",
+  "item.thermal_extra.nectium_coin": "粘液康铜币",
+  "item.thermal_extra.nectium_ingot": "粘液康铜锭",
+  "item.thermal_extra.nectium_dust": "粘液康铜粉",
+  "item.thermal_extra.nectium_gear": "粘液康铜齿轮",
+  "item.thermal_extra.nectium_nugget": "粘液康铜粒",
+  "item.thermal_extra.nectium_plate": "粘液康铜板",
+  "fluid.thermal_extra.polarium": "帕勒瑞恩",
+  "item.thermal_extra.polarium_bucket": "帕勒瑞恩桶",
+  "item.thermal_extra.polarium_coin": "帕勒瑞恩币",
+  "item.thermal_extra.polarium_ingot": "帕勒瑞恩锭",
+  "item.thermal_extra.polarium_dust": "帕勒瑞恩粉",
+  "item.thermal_extra.polarium_gear": "帕勒瑞恩齿轮",
+  "item.thermal_extra.polarium_nugget": "帕勒瑞恩粒",
+  "item.thermal_extra.polarium_plate": "帕勒瑞恩板",
+  "fluid.thermal_extra.zauvium": "扎维姆",
+  "item.thermal_extra.zauvium_bucket": "扎维姆桶",
+  "item.thermal_extra.zauvium_coin": "扎维姆币",
+  "item.thermal_extra.zauvium_ingot": "扎维姆锭",
+  "item.thermal_extra.zauvium_dust": "扎维姆粉",
+  "item.thermal_extra.zauvium_gear": "扎维姆齿轮",
+  "item.thermal_extra.zauvium_nugget": "扎维姆粒",
+  "item.thermal_extra.zauvium_plate": "扎维姆板",
+  "fluid.thermal_extra.drownium": "弱水合金",
+  "item.thermal_extra.drownium_bucket": "弱水合金桶",
+  "item.thermal_extra.drownium_coin": "弱水合金币",
+  "item.thermal_extra.drownium_ingot": "弱水合金锭",
+  "item.thermal_extra.drownium_dust": "弱水合金粉",
+  "item.thermal_extra.drownium_gear": "弱水合金齿轮",
+  "item.thermal_extra.drownium_nugget": "弱水合金粒",
+  "item.thermal_extra.drownium_plate": "弱水合金板",
+  "item.thermal_extra.dragon_breath_plating": "龙息板",
+  "item.thermal_extra.dragon_integral_component": "末影整合组件",
+  "item.thermal_extra.dragon_integral_component.desc": "提升基础属性。其他升级也有可能由此得到进一步增强。",
+  "item.thermal_extra.end_speed_augment": "末影通量链接升级",
+  "item.thermal_extra.end_speed_augment.desc": "提高处理速度但是增加RF消耗。",
+  "item.thermal_extra.ender_rf_coil_augment": "末影RF线圈",
+  "item.thermal_extra.ender_rf_coil_augment.desc": "增加RF的传输，但是降低RF容量。",
+  "item.thermal_extra.nether_fluid_tank_augment": "下界液槽扩容升级",
+  "item.thermal_extra.end_fluid_tank_augment": "末影液槽扩容升级",
+  "item.thermal_extra.cactus_soup": "仙人掌汤",
+  "item.thermal_extra.cactus_juice_bucket": "仙人掌汁叶桶",
+  "fluid.thermal_extra.cactus_juice": "仙人掌汁叶",
+  "itemGroup.thermal_extra": "热力膨胀附加"
```

```suggestion
  "itemGroup.thermal_extra": "热力附加"
```

这份文件的质量不咋地，多处明显错误，我只是粗略看了下，不保证把错找完了